functions/src/export-geojson.spec.ts

```diff
@@ -0,0 +1,201 @@
+import {GroundProtos} from '@ground/proto';
+
+import Pb = GroundProtos.ground.v1beta1;
+const sv = registry.getFieldIds(Pb.Survey);
+const j = registry.getFieldIds(Pb.Job);
+const t = registry.getFieldIds(Pb.Task);
+const l = registry.getFieldIds(Pb.LocationOfInterest);
+const pr = registry.getFieldIds(Pb.LocationOfInterest.Property);
+const p = registry.getFieldIds(Pb.Point);
+const c = registry.getFieldIds(Pb.Coordinates);
+const g = registry.getFieldIds(Pb.Geometry);
+const mq = registry.getFieldIds(Pb.Task.MultipleChoiceQuestion);
+const op = registry.getFieldIds(Pb.Task.MultipleChoiceQuestion.Option);
+
+describe('export()', () => {
+  let mockFirestore: Firestore;
+  const email = 'somebody@test.it';
+  const survey = {
+    [sv.name]: 'Test survey',
+    [sv.acl]: {
+      [email]: SURVEY_ORGANIZER_ROLE,
+    },
+  };
+  const job1 = {
+    id: 'job123',
+    [j.name]: 'Test job',
+    [j.tasks]: [
+      {
+        [t.id]: 'task001',
+        [t.prompt]: 'What is the meaning of life?',
+        [t.textQuestion]: {
+          ['1' /* type */]: Pb.Task.TextQuestion.Type.SHORT_TEXT,
+        },
+      },
+      {
+        [t.id]: 'task002',
+        [t.prompt]: 'How much?',
+        [t.numberQuestion]: {
+          ['1' /* type */]: Pb.Task.NumberQuestion.Type.FLOAT,
+        },
+      },
+      {
+        [t.id]: 'task003',
+        [t.prompt]: 'When?',
+        [t.dateTimeQuestion]: {
+          ['1' /* type */]: Pb.Task.DateTimeQuestion.Type.BOTH_DATE_AND_TIME,
+        },
+      },
+      {
+        [t.id]: 'task004',
+        [t.prompt]: 'Which ones?',
+        [t.multipleChoiceQuestion]: {
+          [mq.type]: Pb.Task.MultipleChoiceQuestion.Type.SELECT_MULTIPLE,
+          [mq.options]: [
+            {
+              [op.id]: 'aaa',
+              [op.index]: 1,
+              [op.label]: 'AAA',
+            },
+            {
+              [op.id]: 'bbb',
+              [op.index]: 2,
+              [op.label]: 'BBB',
+            },
+          ],
+          [mq.hasOtherOption]: true,
+        },
+      },
+      {
+        [t.id]: 'task005',
+        [t.prompt]: 'Where are you now?',
+        [t.captureLocation]: {
+          ['1' /* min_accuracy_meters */]: 999999,
+        },
+      },
+      {
+        [t.id]: 'task006',
+        [t.prompt]: 'Take a photo',
+        [t.takePhoto]: {
+          ['1' /* min_heading_degrees */]: 0,
+          ['2' /* max_heading_degrees */]: 360,
+        },
+      },
+    ],
+  };
+  const pointLoi1 = {
+    id: 'loi100',
+    [l.id]: 'loi100',
+    [l.jobId]: job1.id,
+    [l.customTag]: 'POINT_001',
+    [l.geometry]: {
+      [g.point]: {[p.coordinates]: {[c.latitude]: 10.1, [c.longitude]: 125.6}},
+    },
+    [l.submissionCount]: 0,
+    [l.source]: Pb.LocationOfInterest.Source.IMPORTED,
+    [l.properties]: {
+      name: {[pr.stringValue]: 'Dinagat Islands'},
+      area: {[pr.numericValue]: 3.08},
+    },
+  };
+  const testCases = [
+    {
+      desc: '[GEOJSON] - export point',
```

Comment:
Out of curiosity, what is the "[GEOJSON]" supposed to indicate? That this is exporting to geojson specifically, and that there are going to be other formats? I would probably put these cases in a "describe" block instead, that better captures the category of things being tested.